tests/test_integration.py

```diff
@@ -4,19 +4,19 @@
 from exchange.message import Message
 from exchange.moderators import ContextTruncate
 from exchange.providers import get_provider
-from exchange.providers.ollama import OLLAMA_MODEL
 from exchange.tool import Tool
 from tests.conftest import read_file
 
 too_long_chars = "x" * (2**20 + 1)
 
 cases = [
     # Set seed and temperature for more determinism, to avoid flakes
-    (get_provider("ollama"), os.getenv("OLLAMA_MODEL", OLLAMA_MODEL), dict(seed=3, temperature=0.1)),
+    (get_provider("ollama"), os.getenv("OLLAMA_MODEL", "mistral-nemo"), dict(seed=3, temperature=0.1)),
```

Comment:
The test appears to use qwen2.5:0.5b [as seen here](https://github.com/square/exchange/actions/runs/11096528924/job/30830142869?pr=68#step:8:5).

Was the change to hard-code the ollama model name intentional?

Reply:
wires crossed. agree that even if we want this change, it shouldn't be in the PR about gemini. Personally, I think it is not helpful as the configuration is now tested at least ad-hoc, and removing this constant will lead to drift if we change for example to a llama3.3 or some other capable base choice.

Reply:
Removed the OLLAMA change. Mistake from some issues getting tests to work earlier on.